.env

```diff
@@ -0,0 +1,50 @@
+# Set all your environment variables here
+
+
+#########################################
+## GENERAL
+#########################################
+
+HOSTNAME=localhost
+# Adapt to your production domain and use a reverse proxy
+#HOSTNAME=taiga.company.com
+
+PORT=80
+
+TAIGA_DB_PWD=somethingverysecure,changethis!
+
+
+#########################################
+## SECURITY
+#########################################
+
+TAIGA_SECRET=somethingevenmoresecure!!really,changethis!
+
+TAIGA_ADMIN_PASSWORD=betterthan123123
+
+#########################################
+## EMAIL
+#########################################
+
+TAIGA_EMAIL_DOMAIN=company.com
+TAIGA_SMTP_USER=
+TAIGA_SMTP_PWD=
+
+#########################################
+## LDAP
+#########################################
+
+TAIGA_ENABLE_LDAP=True
+# Adapt to your base domain name
+TAIGA_LDAP_DOMAIN=ldap.company.com
+TAIGA_LDAP_BASE_DN=ou=People,dc=company,dc=com
+
+TAIGA_LDAP_BIND_DN=
+TAIGA_LDAP_BIND_PASSWORD=
```

Comment:
@AminovE99 Setup this to use a local [openldap](https://github.com/osixia/docker-openldap) instance.